Author: Niko Tyni <ntyni@debian.org>
Description: Fix build failures with Perl 5.10.
Pod::Text::pod2text() calling conventions changed
use Newx() instead of New()
--- a/Makefile.PL
+++ b/Makefile.PL
@@ -71,7 +71,7 @@
 	chmod -R u+w,go-w,go+r .
 
 README: src/SpeedyCGI.pm
-	cd src && $(PERL) -e "use Pod::Text; pod2text(-80)" <SpeedyCGI.pm >../README
+	cd src && pod2text -80 <SpeedyCGI.pm >../README
 
 README.html: src/SpeedyCGI.pm
 	cd src && pod2html SpeedyCGI.pm >../README.html && $(RM_F) pod2h*
--- a/src/speedy_backend_main.h
+++ b/src/speedy_backend_main.h
@@ -38,7 +38,7 @@
 
 #else
 
-#define speedy_new(s,n,t)	New(123,s,n,t)
+#define speedy_new(s,n,t)	Newx(s,n,t)
 #define speedy_renew		Renew
 #define speedy_free		Safefree
 
